src/styles/index.css

```diff
@@ -63,3 +63,78 @@
   min-width: 100% !important;
   font-size: 0.9rem;
 }
+
+/* FAQ Page */
+#faq {
+  margin: 2rem;
+}
+.questions {
+  display: block !important;
+  padding: 1em !important;
+}
+.categoryName {
+  display: flex;
+  justify-content: center;
+  font-size: 2em;
+  margin: 1.5em;
+}
+.title {
+  font-size: 1em;
+}
+
+@media (min-width: 720px) {
+  .categoryName {
+    font-size: 4em;
+    margin: 2em;
+  }
+}
+
+@media (min-width: 1440px) {
+  .categoryName {
+    font-size: 6em;
+    margin: 2em;
+  }
+  .questions {
+    width: 80em !important;
+    padding: 2em !important;
+    margin-left: 6em !important;
+  }
+  .category {
+    margin-bottom: 7em !important;
+  }
+  .title {
+    font-size: 2em;
+    padding: 0.5em 0 0.5em 0;
+  }
+  .subTitle {
+    font-size: 1.4em;
+  }
+  .container {
+    font-size: 1.3em;
+  }
+}
+
+@media (min-width: 2560px) {
+  .categoryName {
+    font-size: 10em;
+    margin: 2em;
+  }
+  .questions {
+    width: 160em !important;
```

Comment:
Is there anyway you can avoid !important? It's discouraged and should only be used for last resort.

Reply:
No, because it overwrites geist ui's classes on bases of media queries